src/components/Inflenser/PostCard.js

```diff
@@ -0,0 +1,503 @@
+import { Box, Text, Link, Image, Flex, Avatar, Button, Select, option, useMediaQuery } from "@chakra-ui/react";
```

Comment:
Not really liking anything on this file.

Can you clean it and improve the setup?

Reply:
We are working on the PostCard since we need to get data dynamically, and we are already cleaning some code